meeting-notes/20231207-notes.md

```diff
@@ -0,0 +1,28 @@
+# 07-12-23 Infrastructure Working Group Meeting 10
+
+Attendees:
+- Jim
+- Angus
+- Sarah
+- Ale
+- Brigitta
+
+
+Notes:
+- Help Angus to understand the sense of this meeting
+- What we would like to accomplish 
+- Angus backgroung: using jupyterhub for some time. Experience with Onboarding people ---> Jupyterhub has a barrier if people are not technical 
+    - Identify core offerings from Jupyterhub
```

Comment:
I think all mentions of JupyterHub here are actually meant to be Jupyter _Book_ as that is the project Angus is working on

Reply:
Oh yes, I can see what you mean! Thanks @sgibson91 - did I get all the corrections here?